### PR TITLE
add user profile

### PR DESCRIPTION
**Pull request by Ali-D-Akbar**
# Blog App Frontend:
Description: User Profile Page added having additional (optional) information about the user. Home Page shows an icon of the currently logged user which links to the profile page.

1. run `npm install`
2. run npm start

Screenshot from 2019-10-14 12-23-59
Screenshot from 2019-10-14 12-24-28
Screenshot from 2019-10-14 12-24-42

### CODE REVIEW

File: src/actions/blogList.js [hunk 1]
```diff
@@ -60,10 +60,10 @@ export const getBlogList = (url, order) => (dispatch, getState) => {
                 type: GET_BLOGLIST,
                 payload: res.data
             });
-            dispatch({
-                type: CLEAR_BLOGITEM,
-                payload: null
-            });
+            // dispatch({
```
Comment (zubair-arbi) [inline src/actions/blogList.js]: nit: remove this commented code

File: src/config.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-  "django_server": "https://webblog1234.herokuapp.com",
+  "django_server": "http://127.0.0.1:8000",
```
Comment (zubair-arbi) [inline src/config.json:2]: Do we need to push localhost URL on github?

**Review by zubair-arbi — APPROVED** [review]
👍

File: src/config.json (hunk 2 above)
Comment (Ali-D-Akbar) [reply src/config.json:2]: The frontend uses this variable for API calls. It is the url for backend API.

File: src/actions/blogList.js (hunk 1 above)
Comment (Ali-D-Akbar) [reply src/actions/blogList.js]: Removed.

**Pull request merged**